app/routes/calendar/new.tsx

```diff
@@ -287,14 +287,33 @@ function DescriptionTextarea() {
   );
 }
 
+function getDateWithHoursOffset(date: Date) {
+    const DATE_INPUT_HOURS_OFFSET = 24;
+
+    var copiedDate = new Date(date.getTime());
+    copiedDate.setHours(date.getHours() + DATE_INPUT_HOURS_OFFSET);
+    return copiedDate;
+}
+
 function DatesInput() {
   const { t } = useTranslation(["common", "calendar"]);
   const { eventToEdit } = useLoaderData<typeof loader>();
   const [datesCount, setDatesCount] = React.useState(
     eventToEdit?.startTimes.length ?? 1
   );
-  const isMounted = useIsMounted();
 
+  // React hook that keeps contains an array of parameters that corresponds to each DateInput child object generated
+  //TODO: define a PropType for the object type used in this hook
```

Comment:
I would like to specify a React prop type for the object in the array used by this React Hook, but I'm not sure of the appropriate way to do this.

If anyone can give me some advice, I would appreciate that :) - this should also help resolve the ESLint issue mentioned in this comment: https://github.com/Sendouc/sendou.ink/pull/1062/files#r1008620369

Thank you!

Reply:
This is the type that TS infers:

```ts
{
    finalDateInputDate: Date;
    index: number;
}[]
```

Do you mean you want something different or? :)

Reply:
Thank you Sendou! That helps, I never thought of actually directly copy-pasting what VSCode infers as the type :D 

After I did that, it gave me an error with the correct typing:

`Dispatch<SetStateAction<{ finalDateInputDate: Date; index: number; }[]>>`

I'll just make a new PR with the changes for issue 858, I think what happened with this one is that I accidentally pushed to my forked branch's `rewrite` branch & I didn't revert that before making this new branch off of it